datadog_checks_dev/tests/tooling/configuration/test_load.py

```diff
@@ -2669,37 +2669,40 @@ def test_template_array():
 
     options = spec.data['files'][0]['options'][0]['options']
     option_names = [option['name'] for option in options]
-    assert option_names == [
-        'foo',
-        'proxy',
-        'skip_proxy',
-        'auth_type',
-        'username',
-        'password',
-        'ntlm_domain',
-        'kerberos_auth',
-        'kerberos_delegate',
-        'kerberos_force_initiate',
-        'kerberos_hostname',
-        'kerberos_principal',
-        'kerberos_keytab',
-        'aws_region',
-        'aws_host',
-        'aws_service',
-        'tls_verify',
-        'tls_ignore_warning',
-        'tls_cert',
-        'tls_private_key',
-        'tls_ca_cert',
-        'headers',
-        'extra_headers',
-        'timeout',
-        'connect_timeout',
-        'read_timeout',
-        'log_requests',
-        'persist_connections',
-        'bar',
-    ]
+    assert sorted(option_names) == sorted(
```

Comment:
We care about order to track visual regressions, let's revert this

Reply:
reverted